iphone/TitaniumKit/TitaniumKit/Sources/API/KrollBridge.m

```diff
@@ -34,6 +34,26 @@
 
 @implementation KrollBridge
 
+- (NSDictionary<NSString *, id> *)modules
+{
+  // This is for backwards compatibility with native modules relying on a modules property that
+  // cached module ids -> module instances
+  // Note that this one will only return external native modules and relies on their moduleId
+  // method returning the correct valiue
```

Comment:
Small typo in `valiue`. Also, is the `moduleId` the real ID again here, like `ti.facebook` instead of `TiFacebookModule`?

Reply:
it's whatever the module itself reports from it's `moduleId` method, which should be pre-populated with the correct identifier when the module is created via template.